controller/src/main/java/io/pravega/controller/store/stream/InMemoryStreamMetadataStore.java

```diff
@@ -223,7 +224,17 @@ Scope newScope(final String scopeName) {
     }
 
     @Override
-    public CompletableFuture<Void> addReaderGroupToScope(String scopeName, String rgName, UUID readerGroupId, 
+    public CompletableFuture<Void> addStreamTagsToIndex(String scope, String name, StreamConfiguration config, OperationContext context, Executor executor) {
+        return CompletableFuture.completedFuture(null);
```

Comment:
throw unsupported instead of returning null

Reply:
throwing unsupported causes in memory tests to fail, hence returning a completed future. Let me know if this should be modified.